scripts/doc/create_doc_task_generator.py

```diff
@@ -117,11 +118,7 @@ def main(argv=sys.argv[1:]):
     config = get_config_index(args.config_url)
     index = get_index(config.rosdistro_index_url)
 
-    condition_context = {
-        'ROS_DISTRO': args.rosdistro_name,
-        'ROS_PYTHON_VERSION': index.distributions[args.rosdistro_name].get('python_version'),
-        'ROS_VERSION': 1,
```

Comment:
Here, `ROS_VERSION` is no longer hard coded to `1`, but reflects the version `ROS_DISTRO` refers to.